<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberGui</class>
 <widget class="QWidget" name="MemberGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QComboBox" name="comboBoxDeclarationType"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditType">
     <property name="placeholderText">
      <string>type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditName">
     <property name="placeholderText">
      <string>parameter name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditDefaultValue">
     <property name="placeholderText">
      <string>default</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxGetter">
     <property name="text">
      <string>define getter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxSetter">
     <property name="text">
      <string>define setter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
